cfgov/unprocessed/js/modules/YoutubePlayer.js

```diff
@@ -20,6 +21,8 @@ var API = {
 
   SCRIPT_API: 'http://www.youtube.com/iframe_api',
 
+  IMAGE_URL: 'https://img.youtube.com/vi/%video_id%/maxresdefault.jpg',
```

Comment:
Technically wouldn't be a constant because the id changes. Pretty sure L:86 would break if we updated this to a `const` declaration.